.github/workflows/backport-queue.yml

```diff
@@ -0,0 +1,169 @@
+name: Backport Commit Queue
+
+on:
+  schedule:
+    # Run twice a week at 00:15 AM UTC on Sunday and Thursday.
+    - cron: 15 0 * * 0,4
+  workflow_dispatch:
+
+concurrency: ${{ github.workflow }}
+
+env:
+  NODE_VERSION: lts/*
+  PYTHON_VERSION: '3.12'
+  FLAKY_TESTS: keep_retrying
+  CC: sccache clang
+  CXX: sccache clang++
+  SCCACHE_GHA_ENABLED: 'true'
+
+permissions:
+  contents: read
+
+jobs:
+  commitQueue:
+    runs-on: ubuntu-latest
+    if: github.repository == 'nodejs/node' || github.event_name == 'workflow_dispatch'
+    strategy:
+      fail-fast: false
+      matrix:
+        branch:
+          - { release-line: v23.x, base-branch: main }
+          - { release-line: v22.x, base-branch: v23.x }
```

Comment:
we should be carefule with this because `v23.x` may contain newly released commits that should not be landed during two weeks.

Reply:
I don't think it's a problem, we'll get notified before the commit is ready whether it lands cleanly on v22.x, it doesn't mean we'll have to backport it right away

Reply:
Related to my comment above, I think having the workflow be manually ran makes more sense to start with, in that case these values should be parameterized instead.